webui/src/components/OKR.vue

```diff
@@ -82,11 +82,17 @@
                         </div>
                     </div>
                 </div>
-                <span v-if="!canChangeOkr">
-                    There is no any objective yet. You can send a friendly reminder to your teammate
-                    <span title="Send reminder email"
-                          @click="sendReminder()"><SendIcon/></span>
-                </span>
+                <div v-if="!canChangeOkr">
+                    <span v-if="invalidOneDriveForBusinessLicense">
```

Comment:
Looks like you need to switch v-if and v-else parts

Reply:
I already was in late-night-dev mode =)